app/src/main/java/com/greenfox/kalendaryo/SharingOptionsActivity.java

```diff
@@ -0,0 +1,65 @@
+package com.greenfox.kalendaryo;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+
+import com.greenfox.kalendaryo.adapter.KalendarAdapter;
+import com.greenfox.kalendaryo.adapter.SharingOptionsAdapter;
+import com.greenfox.kalendaryo.http.RetrofitClient;
+import com.greenfox.kalendaryo.http.google.GoogleApi;
+import com.greenfox.kalendaryo.models.KalAuth;
+import com.greenfox.kalendaryo.models.KalMerged;
+import com.greenfox.kalendaryo.models.KalPref;
+import com.greenfox.kalendaryo.models.KalendarsResponse;
+import com.greenfox.kalendaryo.models.SharingOptions;
+
+import java.util.ArrayList;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+
+public class SharingOptionsActivity extends AppCompatActivity {
+
+    private GoogleApi googleApi;
+    private KalPref kalPref;
+    private SharingOptionsAdapter adapter;
+    Button goToChooseAccount;
+    KalMerged kalMerged;
+    RecyclerView recKal;
```

Comment:
@matheb you dont need any class fields, everything is used only in the `onCreate`